.github/workflows/ci.yml

```diff
@@ -10,18 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     
     steps:
-    - run: | 
-          git init azure-sdk-for-java
-          cd azure-sdk-for-java
-          git remote add origin https://github.com/Azure/azure-sdk-for-java.git
-          git config core.sparsecheckout true
-          echo "sdk/spring" >> .git/info/sparse-checkout
-          echo "sdk/jdbc"   >> .git/info/sparse-checkout
-          echo "sdk/spring-experimental" >> .git/info/sparse-checkout
-          echo "eng" >> .git/info/sparse-checkout
-          echo "sdk/boms" >> .git/info/sparse-checkout
-          git pull --depth=1 origin main
-          mvn clean install -Dmaven.javadoc.skip=true -DskipTests \
+    - uses: snow-actions/sparse-checkout@v1.1.0
+      name: Checkout Azure/azure-sdk-for-java
+      with:
+        patterns: |
+          sdk/spring sdk/jdbc sdk/spring-experimental eng sdk/boms
```

Comment:
I know it wasn't there in the first place, but should we add `sdk/parents` as well?

Reply:
I am not very sure but I think it's ok.

Reply:
Updated.